docs/src/main/paradox/additional/osgi.md

```diff
@@ -16,7 +16,7 @@ To use Apache Pekko in OSGi, you must add the following dependency in your proje
 ## Background
 
 [OSGi](https://www.osgi.org/resources/where-to-start/) is a mature packaging and deployment standard for component-based systems. It
-has similar capabilities as [Project Jigsaw](https://openjdk.java.net/projects/jigsaw/) (originally scheduled for JDK 1.8), but has far stronger facilities to
```

Comment:
this link seem likes has been change.

Reply:
`https://openjdk.org/projects/jigsaw/` I can access it.

Maybe should drop osgi, as I just used it once ten years ago.